Components/Core/Inc/Timer.hpp

```diff
@@ -28,7 +35,10 @@ class Timer
 	Timer();
 
 	bool ChangePeriod(const uint32_t period);
-
+	bool StartTimer();
+	bool StopTimer();
+	CurrentState GetState(TimerHandle_t xTimer);
+	CurrentState completed(TimerHandle_t xTimer);
```

Comment:
Shouldn't need this, if we do then capitalize it.

If it's meant to be used as a callback, it'll have to be static.

Reply:
Did you mean we don't need the completed function or all of them?
In the case of the completed function, I altered the code so I am using states instead of the completed function.

Reply:
Just the completed function